components/inputs/input-checkbox.js

```diff
@@ -123,7 +125,6 @@ class InputCheckbox extends InputInlineHelpMixin(FocusMixin(SkeletonMixin(LitEle
 			css`
 				:host {
 					display: block;
-					line-height: ${cssSizes.inputBoxSize}rem;
```

Comment:
Because this was applied to the host, it was also getting inherited by the supporting slot. That deviated from the standard `1.2rem` line height from our typography styles. Applying it to the `label` only accomplishes the same thing.